config/config.proto

```diff
@@ -70,6 +70,17 @@ message SlackConfig {
   optional bool send_resolved = 5 [default = false];
 }
 
+message FlowdockConfig {
+  // Flowdock flow API token
+  optional string api_token = 2;
+  // Flowdock from_address
+  optional string from_address = 3;
+  // Flowdock flow tags
+  repeated string tags = 4;
```

Comment:
You'll probably want to call this just `tag`, because when using a repeated field, you just list it multiple times with as the same field name:

```
tag: "foo"
tag: "bar"
...
```